### ISSUE TITLE
Feature request - Add option to delete empty folders after moving item

### ISSUE THREAD

**Issue by MarcioFCMartins** [wileyyugioh/zotmoov#42]
Thank you for this addon, it's simple and does something that I really need to organize my library.

However, I 'd like to see an option to automatically delete folders that are empty after moving attachments.

**Comment by tanjiu** [comment]
是的，同步盘里面日久累积了好多空文件夹，能否添加一个删除条目的功能，实测从zotero中删除条目，同步盘里面的条目并不会被删除

**Comment by tanjiu** [comment]
> Which folders do you mean? If you mean the folders in Zotero storage, although they appear empty they actually contain cached text.

<img width="494" alt="dc1e8c814c3c0d630a3fbbe6158b042" src="https://github.com/user-attachments/assets/b7a89350-2cab-4ee4-94c5-fdc966c218f5">
<img width="311" alt="7b4a8208dc097a54ed07316b6da1b1c" src="https://github.com/user-attachments/assets/220a1fdf-9d06-492a-8c5f-6d1e1ec8a69b">

如上图所示，我新建了一个类别并向里面添加了一个附件，我的同步云盘里会出现对应的新建文件夹，但是如果我选择删除该条目及附近，该文件夹内的文件会被删除，但是该文件夹还会存在，即使我清空了zotero里面的回收站。

<img width="533" alt="278dfe883ee646c5cae6d69da908485" src="https://github.com/user-attachments/assets/56573121-f997-4e96-913a-8b20178b8537">
![1732032493426](https://github.com/user-attachments/assets/7823f173-0849-4977-b57d-a219338c1d14)

zotero本身的存储文件夹内确实会有一些空文件夹，但是同步盘内的文件夹好像并不能被选择真正删除，能否增加一个删除同步盘里面空文件夹的功能？因为我发现即使在zotero里面新建一个分类，如果该类别里面没有传入文件的话就不会创建该文件夹，只有传入过一次文件以后才会创建，但是删除的时候又不会被真正删除，我认为可以改进一下。

**Comment by wileyyugioh** [comment]
@tanjiu Did you check [the option](https://github.com/wileyyugioh/zotmoov/blob/master/docs/SETTINGS_INFO.md#automatically-delete-external-linked-files-in-the-zotmoov-directory)?

**Comment by tanjiu** [comment]
> @tanjiu Did you check [the option](https://github.com/wileyyugioh/zotmoov/blob/master/docs/SETTINGS_INFO.md#automatically-delete-external-linked-files-in-the-zotmoov-directory)? Also can you see if there are any hidden files in the folder?

这是我的zotmoov的设置，应该没有什么问题，添加和删除文件时都可以正常工作，只有删除类别时会出现问题
<img width="783" alt="6c2f7b34f70057630621a28ea0b0b50" src="https://github.com/user-attachments/assets/a8a535de-887e-45aa-a34a-be9eca923a2b">

**Comment by tanjiu** [comment]
> @tanjiu Did you check [the option](https://github.com/wileyyugioh/zotmoov/blob/master/docs/SETTINGS_INFO.md#automatically-delete-external-linked-files-in-the-zotmoov-directory)? Also can you see if there are any hidden files in the folder?

是我的问题，谢谢

**Comment by MakitaYouki** [comment]
I agree. For example, when I decide to create a new folder to store items, the old one  won't be deleted by zotmoov, instead I need to delete it manually.
